<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Replacements</class>
 <widget class="QDialog" name="Replacements">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Define replacement</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="okCancel_buttons">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Replace:</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Thunk:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Function:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="libToReplaceLabel">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="thunkLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>130</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>With:</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <property name="verticalSpacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>45</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>45</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Library:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="libraryEdit">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>30</y>
          <width>284</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>example.dll</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="functionEdit">
        <property name="geometry">
         <rect>
          <x>65</x>
          <y>60</y>
          <width>284</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>functionName</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Choose</string>
       </attribute>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>50</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Library:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="functionCombo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>62</y>
          <width>239</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="libraryCombo">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>31</y>
          <width>239</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addLibraryButton">
        <property name="geometry">
         <rect>
          <x>319</x>
          <y>30</y>
          <width>30</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>62</y>
          <width>50</width>
          <height>20</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Function:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okCancel_buttons</tabstop>
  <tabstop>thunkLabel</tabstop>
  <tabstop>libToReplaceLabel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
